deploy/monitoring/bin/setup-server.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+ansible-playbook ./setup-server.yml -i ./hosts "$@"
```

Comment:
I don't think we need setup-server script in the grafana, as server is set up as part of the project deploy.